src/views/index.jade

```diff
@@ -163,7 +163,8 @@ block content
             div.active-nodes.text-warning
               i.icon-warning-o
               span.small-title ATTENTION!
-              span.small-value This page does not represent the entire state of the Ellaism network - listing a node on this page is a voluntary process.
+              span.small-value A fork will take place at block 2 000 000. Support for other clients will be deprecated. Please use Parity v.1.10.3+
```

Comment:
Let's be a little bit more specific there. `other clients` -> `multi-geth and go-ellaism`.